src/core/quicdef.h

```diff
@@ -128,6 +128,12 @@ typedef struct QUIC_PATH QUIC_PATH;
 //
 #define QUIC_DEFAULT_RETRY_MEMORY_FRACTION      65 // ~0.1%
 
+//
+// If enabled, workers will poll the number of times before falling back to the
+// wait or delay state.
+//
+//#define QUIC_WORKER_POLLING                     10000
```

Comment:
Disabled for now, but shows promise in DPDK/XDP scenarios. I want to get this all in mainline before merging into dpdk branch to play around with this part some more.